features/step_definitions/collection.js

```diff
@@ -55,12 +55,12 @@ Before('@collections', function hook() {
     this.repoOrg = this.testOrg;
     this.repoName = 'functional-collections';
     this.pipelineId = null;
-    this.firstCollectionId = null;
-    this.secondCollectionId = null;
+
+    this.collections = {};
 });
 
 When(/^they check the default collection$/, { timeout: TIMEOUT }, function step() {
-    return this.ensurePipelineExists({ repoName: this.repoName }).then(() =>
+    return this.ensurePipelineExists({ repoName: this.repoName, shouldNotDeletePipeline: true }).then(() =>
```

Comment:
would this `shouldNotDeletePipeline` not conflict on second run?

Reply:
no, I have tested it locally.